### ISSUE TITLE
Running YCSB tests on multinode Cassandra

### ISSUE THREAD

**Issue by swaapnika-guntaka** [brianfrankcooper/YCSB#1024]
I'm trying to benchmark Cassandra on a multi node cluster. Unlike other distributed databases, in Cassandra there is no concept of master/slave. Should I mention all the Cassandra hosts in the  -hosts in the command to execute it against the cluster?

**Comment by busbey** [comment]
Yep! Would the README be clearer if we expressly called this out?

**Comment by swaapnika-guntaka** [comment]
Probably yes.. This helps to understand the process of the loading and running the workload in detail. Thanks. This issue can be closed now
